src/server/middleware/authentication.js

```diff
@@ -20,10 +19,8 @@ function parseHeader(header) {
 }
 
 function jwtVerify(req, res, header, done) {
-  console.log(`Secret ${jwtSecret}`);
-  console.log(`Audience ${jwtAudience}`);
   const token = parseHeader(header);
-  jwt.verify(token, jwtSecret, { aud: jwtAudience }, (err, decoded) => {
+  jwt.verify(token, jwtSecret, (err, decoded) => {
```

Comment:
Yes!! Exactly.
Either that or it needs to be included in payload as aud.
So after merging this we I will create a token with aud key in the payload .